### PR TITLE
feat: refactored club list page

### PR DESCRIPTION
**Pull request by aomi**
This PR creates new Chakra-UI focused page/components for the club list page and club info page. 
Links within the app are generated centrally from a `links.ts` file.

The `ng` prefix will be removed once QA is finished.

### CODE REVIEW

**Review by markksr5 — APPROVED** [review]

**Comment by markksr5** [comment]
You'll remove the ng in the url before merge right?

**Comment by aomi** [comment]
@markksr5 yup :+1:

**Pull request merged**
